src/store/reducers/drinks.js

```diff
@@ -21,7 +22,10 @@ const setDrinks = (state, action) => {
   return {
     ...state,
     loading: false,
-    drinks: results,
+    drinks: [
+      ...state.drinks,
```

Comment:
I think the first spread is not necessary

Reply:
It is necessary, otherwise the old state is replaced by the new results.
Tried to removed and didn't work.